internal/app/api/v1/scripts.go

```diff
@@ -131,10 +132,10 @@ func (s kubtestAPI) ExecuteScript() fiber.Handler {
 		}
 
 		// get executor from kubernetes CRs
-		executor, err := s.Executors.Get(scriptCR.Spec.Type_)
-		if err != nil {
-			return s.Error(c, http.StatusInternalServerError, fmt.Errorf("can't get executor: %w", err))
-		}
+		// executor, err := s.Executors.Get(scriptCR.Spec.Type_)
```

Comment:
I would leave job based executor/runner to be registered as executor/runner ? What do you think ?

Reply:
Literally no need for it now, we could perhaps recycle it so we can get appropriate image for the pod

Reply:
> Recent comments have reminded me that we need to figure out the way to "watch" pods the same way it was done by "watching" mongodb for changes

even better - http watch simply was asking in loop for get and checked for status change (success|error)
here we can easily implement watch based on logs (just need to tag those lines which are logs somehow